src/ci.js

```diff
@@ -20,6 +31,20 @@ class CI {
     } else {
       return(this.generic())
     }
+
+  analytics_env() {
+    return this._stripUndefinedKeys({
+      "key": process.env.BUILDKITE_ANALYTICS_KEY,
+      "url": process.env.BUILDKITE_ANALYTICS_URL,
+      "branch": process.env.BUILDKITE_ANALYTICS_BRANCH,
+      "commit_sha": process.env.BUILDKITE_ANALYTICS_SHA,
+      "number": process.env.BUILDKITE_ANALYTICS_NUMBER,
+      "job_id": process.env.BUILDKITE_ANALYTICS_JOB_ID,
+      "message": process.env.BUILDKITE_ANANLYTICS_MESSAGE,
```

Comment:
```suggestion
      "message": process.env.BUILDKITE_ANALYTICS_MESSAGE,
```

Reply:
OH that typo is also in the rspec collector!!!

Reply:
🤦